communitiesuk/funding-service-design-assessment-store
# Fs 2501 automate import of submitted applications from application into assessment

https://dluhcdigital.atlassian.net/browse/FS-2501

### Description
This PR is a part of the work for FS-2501

- Implemented SQS receive & delete functions to process submitted queues.
- Added unit tests for SQS helpers functions
- Implemented Scheduler to run the receive/delete message functions periodically.
- Refactored code to bulk insert function to accommodate changes

linked PR https://github.com/communitiesuk/funding-service-design-application-store/pull/207

### How to test
- Pull the changes locally and also from [application-store](https://github.com/communitiesuk/funding-service-design-application-store/pull/207) and docker-runner
- rebuild the docker containers.
- Go to frontend & submit an application
- Observe that application is automatically imported to assessment in 60 seconds time
